Topic0_2/src/SQLFactory/Demo.java

```diff
@@ -0,0 +1,12 @@
+package SQLFactory;
+
+import java.sql.Connection;
+
+public class Demo {
+	public static void main (String[] args) {
+		AbstractFactory abstractfactory = new AbstractFactory();
+		SQLType SQLT = new MySQLType();
+		Connection connecting = SQLT.getConnection("MySQL");
```

Comment:
Typically in Java we use UPPER_CASE variables when they are constants.

Reply:
I will keep it in mind, thanks